### DISCUSSION TITLE
Low speeds in `get.gravitational.com`

### DISCUSSION THREAD

**Discussion by xlejo** [gravitational/teleport#14638] [General]
Hi, Teleport brains!

First, thanks to all collaborators and maintainers for the amazing project!

Now my problem :). Sometimes I download the binary of Teleport (any version) with a speed of 200KB/s (kilobytes) in a symmetric fiber connection of 500MB (megabits).

It's normal? Maybe the absence of a CDN (content delivery network) or mirror repository for `get.gravitational.com` cause this problem?

I currently live in Argentina.

**Comment by webvictim** [comment]
Unfortunately, yes, this is normal. I have the same problem in Canada on a gigabit connection. It's definitely an issue with the lack of a CDN and all traffic going via the GCP network through a fixed ingress point in one region.

I'm hoping we'll have this fixed soon, but until then, please be patient!

**Reply by xlejo** [reply]
That make sense.

Thanks for the reply!
